Added Document Merge as Open Source Connector

cirrussoft opened the pull request:
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to a high volume of connector submissions. It may take 1-2 weeks for us to provide comments for your connector.

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

cirrussoft commented on certified-connectors/Document Merge/readme.md:
```diff
@@ -0,0 +1,26 @@
+﻿
```
@Amjed-Ayoub  We have updated readme file as requested, kindly verify same and do let us know in case anything is missing

Amjed-Ayoub replied:
Hi @cirrussoft, thank you for the update, the readme is missing required fields. please see below.
1. Publisher: Publisher's Name/Required. It should be the company or organization name.
2. Known Issues and Limitations/Required. Include any known issues and limitations a user may encounter. if there are no issues, please mention that "No known issues or limitations".
Thank you very much.

cirrussoft commented:
This has been updated. Please check and confirm.

Thanks,


From: Amjed Ayoub ***@***.***>
Sent: 19 April 2022 23:30
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: CIRRUSSOFT LTD ***@***.***>; Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] Added Document Merge as Open Source Connector (PR #1573)


@Amjed-Ayoub commented on this pull request.

________________________________

In certified-connectors/Document Merge/readme.md<https://github.com/microsoft/PowerPlatformConnectors/pull/1573#discussion_r853470942>:

> @@ -0,0 +1,26 @@



+﻿



Hi @cirrussoft<https://github.com/cirrussoft>, Thank you for open sourcing your connector. can you please make sure to include all required fields in the readme file? You can follow this template<https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors>.
Thank you very much.

—
Reply to this email directly, view it on GitHub<https://github.com/microsoft/PowerPlatformConnectors/pull/1573#pullrequestreview-946273477>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AMWO37QOQ3V33VCQDJFDJ6TVF4XV5ANCNFSM5TLGXTLQ>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

cirrussoft commented:
@Amjed-Ayoub  Could you please revalidate the readme.md file, updated it as requested

cirrussoft commented:
@Amjed-Ayoub  Is there any update on this connector for validation check?

sriyen-msft commented:
@cirrussoft **Are you sure you want to delete 1592 files from Microsoft open-source rep with this PR?**

cirrussoft commented:
> o delete 1592 files from Microsoft open-source rep with this PR?

Yes Please

sriyen-msft commented:
> > o delete 1592 files from Microsoft open-source rep with this PR?
> 
> Yes, please, we have removed those files from our local repository from GitHub

@cirrussoft If you want to remove the files from your repository, you can raise a PR directly into your repository and get it approved and removed by merging into it. But, please let us know the reason to raise this PR to remove all these files in Microsoft's open-source repository?

cirrussoft commented:
> > > o delete 1592 files from Microsoft open-source rep with this PR?
> > 
> > 
> > Yes, please, we have removed those files from our local repository from GitHub
> 
> @cirrussoft If you want to remove the files from your repository, you can raise a PR directly into your repository and get it approved and removed by merging into it. But, please let us know the reason to raise this PR to remove all these files in Microsoft's open-source repository?

No, those are removed from our local repo, not from MS repo

sriyen-msft commented:
> > > > o delete 1592 files from Microsoft open-source rep with this PR?
> > > 
> > > 
> > > Yes, please, we have removed those files from our local repository from GitHub
> > 
> > 
> > @cirrussoft If you want to remove the files from your repository, you can raise a PR directly into your repository and get it approved and removed by merging into it. But, please let us know the reason to raise this PR to remove all these files in Microsoft's open-source repository?
> 
> No, those are removed from our local repo, not from MS repo

@cirrussoft Please make sure that you have created the fork based on the "dev" branch of our [repository](https://github.com/microsoft/PowerPlatformConnectors). When you have a fork, please note that it is a replica of Microsoft's open-source repository, and whatever you make changes into the fork, the changes would be applied to our Microsoft's open-source repo when you raise a PR like this. Please go thru the documentation and let me know if you need any assistance from our team, if required, we can schedule a meeting with you in PST time. Thanks

Amjed-Ayoub commented:
Hi @cirrussoft, I hope you are doing well. I am following up with you to see how things are going with updating your pull request and remove other connector files.
Please let us know if you have any questions.
We look forward to getting your connector certified.
Thank you very much.

cirrussoft commented:
> Hi @cirrussoft, I hope you are doing well. I am following up with you to see how things are going with updating your pull request and remove other connector files. Please let us know if you have any questions. We look forward to getting your connector certified. Thank you very much.

Dear @Amjed-Ayoub Please go ahead and merge the changes
